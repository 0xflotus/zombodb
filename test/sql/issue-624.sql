create table issue624 (title text);
insert into issue624 select convert_from(decode('QXRlbGllcnMgZOKAmcOpY2hhbmdlIGV0IGRlIHByYXRpcXVlIHBvdXIgC2xlcyBtYW5hZ2VycyBvcMOpcmF0aW9ubmVscw==', 'base64'), 'UTF8');
select * from issue624;
create index idxissue624 on issue624 using zombodb ((issue624.*));
select * from zdb.terms('issue624', 'title', '') order by term;
drop table issue624;