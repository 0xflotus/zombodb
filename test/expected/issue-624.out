create table issue624 (title text);
insert into issue624 select convert_from(decode('QXRlbGllcnMgZOKAmcOpY2hhbmdlIGV0IGRlIHByYXRpcXVlIHBvdXIgC2xlcyBtYW5hZ2VycyBvcMOpcmF0aW9ubmVscw==', 'base64'), 'UTF8');
select * from issue624;
                                 title                                 
-----------------------------------------------------------------------
 Ateliers d’échange et de pratique pour \x0Bles managers opérationnels
(1 row)

create index idxissue624 on issue624 using zombodb ((issue624.*));
select * from zdb.terms('issue624', 'title', '') order by term;
     term      | doc_count 
---------------+-----------
 ateliers      |         1
 de            |         1
 d’échange     |         1
 et            |         1
 les           |         1
 managers      |         1
 opérationnels |         1
 pour          |         1
 pratique      |         1
(9 rows)

drop table issue624;
